src/components/NcActionCheckbox/NcActionCheckbox.vue

```diff
@@ -206,8 +206,7 @@ export default {
 
 		&::before {
 			margin-block: 0 !important;
-			// 16px is the width of the checkbox including the border
-			margin-inline: calc((var(--default-clickable-area) - 16px) / 2) !important;
+			margin-inline: calc((var(--default-clickable-area) - 14px) / 2) !important;
```

Comment:
Probably, we should either add CSS variable for it, or change layout here to not use this number to avoid magic numbers.

P.S. On all supported versions on server it is 14px, not 16px

https://github.com/nextcloud/server/blob/stable28/core/css/inputs.scss#L414